tmt/steps/prepare/feature.py

```diff
@@ -0,0 +1,269 @@
+            # yum install epel-release
+            self.info('Enable EPEL on CentOS 7')
+            shscript1 = ShellScript(f'{sudo} yum -y install {" ".join(CENTOS_7_PACKAGES)}')
+            self.guest.execute(shscript1, silent=True)
+        elif guest_distro == Distro.CENTOS_STREAM_8:
+            # dnf config-manager --set-enabled powertools
+            # dnf -y install epel-release epel-next-release
+            self.info('Enable EPEL on CentOS Stream 8')
+            shscript1 = ShellScript(f'{sudo} dnf config-manager --set-enabled {FEDORA_REPO}')
+            shscript2 = ShellScript(f'{sudo} dnf -y install {" ".join(FEDORA_PACKAGES)}')
+            self.guest.execute(shscript1 & shscript2, silent=True)
+        elif guest_distro == Distro.CENTOS_STREAM_9:
+            # dnf config-manager --set-enabled crb
+            # dnf -y install epel-release epel-next-release
+            self.info('Enable EPEL on CentOS Stream 9')
+            shscript1 = ShellScript(
+                f'{sudo} dnf config-manager --set-enabled {CENTOS_STREAM_9_REPO}')
+            shscript2 = ShellScript(f'{sudo} dnf -y install {" ".join(FEDORA_PACKAGES)}')
+            self.guest.execute(shscript1 & shscript2, silent=True)
+        elif guest_distro == Distro.RHEL_7:
+            # subscription-manager repos --enable rhel-*-optional-rpms \
+            #               --enable rhel-*-extras-rpms \
+            #               --enable rhel-ha-for-rhel-*-server-rpms
+            # yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+            self.info('Enable EPEL on RHEL 7')
+            shscript1 = ShellScript(f'{sudo} subscription-manager repos --enable {RHEL_7_REPO}')
+            shscript2 = ShellScript(f'{sudo} yum -y install {" ".join(RHEL_7_PACKAGES)}')
+            self.guest.execute(shscript1 + shscript2, silent=True)
+        elif guest_distro == Distro.RHEL_8:
+            # subscription-manager repos --enable codeready-builder-for-rhel-8-$(arch)-rpms
+            # dnf -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+            self.info('Enable EPEL on RHEL 8')
+            shscript1 = ShellScript(f'{sudo} subscription-manager repos --enable {RHEL_8_REPO}')
+            shscript2 = ShellScript(f'{sudo} dnf -y install {" ".join(RHEL_8_PACKAGES)}')
+            self.guest.execute(shscript1 + shscript2, silent=True)
+        elif guest_distro == Distro.RHEL_9:
+            # subscription-manager repos --enable codeready-builder-for-rhel-9-$(arch)-rpms
+            # dnf -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm
+            self.info('Enable EPEL on RHEL 9')
+            shscript1 = ShellScript(f'{sudo} subscription-manager repos --enable {RHEL_9_REPO}')
+            shscript2 = ShellScript(f'{sudo} dnf -y install {" ".join(RHEL_9_PACKAGES)}')
+            self.guest.execute(shscript1 + shscript2, silent=True)
+        else:
+            pass
```

Comment:
Silence is not an option :) Here should be at least a warning about the unsupported distro.

Reply:
Added.